src/Components/HomePage/ElementsContainer.js

```diff
@@ -1,86 +1,74 @@
-import React, { Suspense } from 'react';
+import React/*, { Suspense } */ from 'react';
+import { Link } from 'react-router-dom';
 
-import { BannerContainer, CategoriesContainer, ProductContainer, SectionHeader } from './styled';
+import { CategoriesContainer, ProductContainer, SectionHeader } from './styled';
 
-const Slider = React.lazy(() => import('../Slider/Slider'));
-const ProductElement = React.lazy(() => import('./ProductElement'));
-const CategoriesElement = React.lazy(() => import('./CategoriesElement'));
-const Button = React.lazy(() => import('Components/Button'));
+import ProductElement from './ProductElement';
+import CategoriesElement from './CategoriesElement';
+import Button from 'Components/Button';
 
-const ElementsContainer = (props) => {
-    //const [fetchedData, setFetchedData] = React.useState(null);
-    
-    //TODO: Check if its really nedded: <Error boundarys />
-    //const [error, setError] = React.useState(null);
+import { useFeaturedBanners } from 'utils/hooks/useFeaturedBanners';
 
-    //TODO: Check if its really needed:  fallback={}
-    //const [isLoading, setIsLoading] = React.useState(false);
-    
-    /* React.useEffect(() => {
-        setIsLoading(true);
-        
-        setError(null);
-        setIsLoading(false);
-    }, []); */
+//const Slider = React.lazy(() => import('../Slider/Slider'));
+//const ProductElement = React.lazy(() => import('./ProductElement'));
+//const CategoriesElement = React.lazy(() => import('./CategoriesElement'));
+//const Button = React.lazy(() => import('Components/Button'));
```

Comment:
If you won't be needed these comments you should remove them from the code. If you are planing on reusing them, it is better to add a `TODO`.